overlays/DIMO/dimo.vspec

```diff
@@ -47,8 +47,14 @@ Vehicle.DIMO.VehicleID:
   datatype: string
   description: unique DIMO ID for the vehicle
 
-Vehicle.DIMO.IsLocationRedacted:
+Vehicle.CurrentLocation.IsRedacted:
   datatype: boolean
   type: sensor
   description: Indicates if the latitude and longitude signals at the current
                timestamp have been redacted using a privacy zone.
+
+Vehicle.Powertrain.CombustionEngine.EngineOilRelativeLevel:
```

Comment:
It wouldn’t hurt to give min/max like they do, just to reinforce the range. I understand that it’s float because we don’t care, that part is fine

https://github.com/COVESA/vehicle_signal_specification/blob/5b677591eb226e2f840cd53fc5051326c01966f7/spec/Powertrain/FuelSystem.vspec#L49

Reply:
Will update and then merge